### DISCUSSION TITLE
Future of Serverpod

### DISCUSSION THREAD

**Discussion by mbnoimi** [serverpod/serverpod#125] [General]
I'm a bit curious about the future of Serverpod although I didn't use it at all. Historical there are many backend frameworks written in Dart became dead right now although most of them didn't live more than 5-7 years. 

I don't know why this frequently happens with Dart based backend frameworks. May the creator of this framework (@vlidholt) explain to the community why Serverpod is different from these backend frameworks (specifically [Angel](https://github.com/angel-dart/angel) and [Aqueduct](https://github.com/stablekernel/aqueduct))

I came from C++ background so I found Dart syntax very familiar to me by comparing to Python. I like the way that I may write my backend & frontend using the same language but I became very hesitated about this way after reading in Dart  backend frameworks history.

Currently I'm using [FastAPI](https://github.com/tiangolo/fastapi) as a reliable backend but I didn't drop my dream in using Dart in backend and frontend.

May you please give us your point of view specifically for the hesitators (I read many Aqueduct users explained there anger of dropping Aqueduct development some people went to [Shelf](https://github.com/dart-lang/shelf) while others left Dart for good)

**Comment by MarkOSullivan94** [comment]
Whilst Aqueduct has been sunset, [Conduit](https://www.theconduit.dev/) forked Aqueduct and is an active fork of it.

My hope is that both Serverpod and Conduit will be around for a long time but perhaps more importantly it'd be good to have a community for those passionate about using Dart on the backend.

Maintaining open source software is tiring and unrewarding, I hope that companies using Serverpod or Conduit will sponsor both of the projects and ensure that there's enough financial incentive for the maintainers to keep working on them.

There's an unofficial Dart Discord server for Dart developers to gather which I'm hoping will be better than Gitter which seemed a bit dead.

Here's the Reddit post where I found out about the Discord: https://www.reddit.com/r/dartlang/comments/nzvmmh/come_discuss_dart_on_the_unofficial_dart/

Discord server:  https://discord.gg/Qt6DgfAWWx

**Reply by yahu1031** [reply]
Here is [Flutter project discord server](https://discord.gg/BS8KZyg) - Where generally flutter/dart contributors generally talk. The Flutter Reddit(official) [discord server](https://discord.gg/rflutterdev)

**Comment by vlidholt** [comment]
I took the freedom to convert this into a discussion instead to keep things a bit more tidy.

I think one of the past reasons that previous frameworks hasn't worked out too well is that they were made before Flutter. With Flutter, the Dart community is growing very fast and there is a much higher demand for Dart on the server side too. Serverpod is also specifically made for Flutter, which makes it faster and better to work with than traditional web servers.

We are planning to support Serverpod for a long time in the future and to build a sustainable business around it.

I'll check out that Discord server, thanks for the tip!

**Reply by mbnoimi** [reply]
I was planning to post a bunch of other questions but [this interview](https://www.youtube.com/watch?v=79O9w6R4CmA) answered most of my questions.

The most important summary I got is Serverpod has a business model which will make it likely reliable in the near future (hosting plans within some clicks). This is very awesome and make it so different from the historical Dart backend projects.

I'll break my questions into other threads because the most important points already discussed in the interview **BUT** I want to suggest to take the economic plan in the consideration because [SuperBase](https://supabase.com/) already competing Firebase in the cost.

**Reply by jfuqua7** [reply]
appWrite is directly targeting Firebase.
